spec/unit/mutant/matcher/methods/instance_spec.rb

```diff
@@ -107,9 +107,9 @@ def apply
 
       expect(capture_reporter.warnings).to eql([<<~'MESSAGE' % { scope: scope, exception: exception }])
         Caught an exception while accessing a method with
-        #instance_method that is part of #{public,privat,protected}_instance_methods.
+        #instance_method that is part of #{public,private,protected}_instance_methods.
 
-        This is a bug in your ruby implementation its stdlib, libaries our your code.
+        This is a bug in your ruby implementation, its stdlib, other dependencies, or your code.
```

Comment:
not sure this is exactly what you meant but i thought "other dependencies" might be clearer.

`libaries` and `our` were both typos

Reply:
note: changed other -> your as well